factorial_02.py

```diff
@@ -0,0 +1,10 @@
+num1=input("Enter number: ")
+try:
```

Comment:
Please explain what is 'try' in multiline comment at the end or start of program

Reply:
#The try block lets you test a block of code for errors.

Reply:
#The try block lets you test a block of code for errors.